toolkit/tools/imagegen/configuration/parse_partition_test.go

```diff
@@ -18,7 +18,7 @@ var (
 	ValidPartitionCommand2 = "part biosboot --fstype=biosboot --size=8 --ondisk=/dev/sda"
 
 	validTestParserDisk_OnePartition = Disk{
-		PartitionTableType: "gpt",
+		PartitionTableType: "mbr",
```

Comment:
Yes, this change is just to ensure that the unit tests will pass now given that in this mode the default partition table type is MBR instead of GPT. Will add new test case to handle testing GPT.

Reply:
Were these new test cases added for handling GPT?

Reply:
I looked into adding new test cases to handle GPT but looks like it can't be done since /proc/cmdline is read-only and thus I couldn't modify it to inject --gpt=True to simulate the scenario.

I'm intending to add test case for GPT in another PR which adds firmware check support in the parser module.